src/ggrc/notifications/common.py

```diff
@@ -207,19 +215,84 @@ def sort_comments(notif_data):
     comment_notifs[parent_obj_info] = comments_as_list
 
 
-def get_pending_notifications():
+def filter_cycle_tasks_caches(notifications, tasks_caches):
+  """Filter related tasks by notification id"""
+  task_ids = [n.object_id for n in notifications
+              if n.object_type == "CycleTaskGroupObjectTask"]
+
+  return {i: tasks_caches[i] for i in task_ids if i in tasks_caches}
+
+
+def split_notif_by_chunks(notifications, chunk_size=7000):
+  """Split notification by chunks.
+
+  Args:
+    notifications (list of Notification): List of notification.
+    chunk_size (int): chunk size.
+  """
+  result = []
+  count_chunks = len(notifications) // chunk_size + 1
+  start_index = 0
+  for i in range(start_index, count_chunks + 1):
+    stop_index = start_index + chunk_size
+    result.append(notifications[start_index:stop_index])
+    start_index = i * chunk_size
+  return result
+
+
+def build_pagination_params():
+  """Build pagination params from request obj
+
+  :return:
```

Comment:
According to our style guide it should be
```suggestion
  Returns:
```

Reply:
Fixed.